admin/pre-reg/templates/edit_draft_registration.html

```diff
@@ -0,0 +1,103 @@
+{% extends 'base.html' %}
+{% load render_bundle from webpack_loader %}
```

Comment:
Second PR. Also, do we want to consider mako templates? https://www.djangopackages.com/packages/p/django-mako/